src/utils/webrtc/webrtc.js

```diff
@@ -1613,27 +1613,18 @@ export default function initWebRtc(signaling, _callParticipantCollection, _local
 	})
 
 	webrtc.on('sendToAll', function(messageType, payload) {
-		const sessionIdsForParticipantsWithPeers = {}
-		webrtc.webrtc.peers.forEach(peer => {
-			sessionIdsForParticipantsWithPeers[peer.id] = peer
-		})
-
-		// "webrtc.sendToAll" only sends the signaling message to participants
-		// for which there is a Peer object. Therefore the message needs to be
-		// explicitly sent here too to participants without audio and video.
 		for (const sessionId in usersInCallMapping) {
-			if (sessionIdsForParticipantsWithPeers[sessionId]) {
-				continue
-			} else if (!usersInCallMapping[sessionId].inCall) {
+			if (!usersInCallMapping[sessionId].inCall) {
```

Comment:
Might be a problem for typingIndicators, because we are not in a call usually, but fine for this case.

Reply:
This code is used only for calls, [typing indicators are sent in a different way](https://github.com/nextcloud/spreed/blob/9d270d833a66ff877200e21aec1fd6975ba9992e/src/utils/SignalingTypingHandler.js#L99-L108) so this should be fine :-)

Thanks nevertheless for the warning!